policyengine_us/variables/gov/states/oh/tax/income/credits/oh_earned_income_credit.py

```diff
@@ -0,0 +1,20 @@
+from policyengine_us.model_api import *
+
+
+# This credit is Non-refundable
+class oh_earned_income_credit(Variable):
```

Comment:
For consistency with other states' EITC matches
```suggestion
class oh_eitc(Variable):
```